monai/transforms/composables.py

```diff
@@ -86,17 +81,10 @@ def __call__(self, data):
         d = dict(data)
         for key, interp in zip(self.keys, self.interp_order):
             affine_key = self.meta_key_format.format(key, 'affine')
-            original_key = self.meta_key_format.format(key, 'original_affine')
-            affine = d.get(affine_key, None)
-            if affine is None:
-                affine = d.get(original_key, None)
             # resample array of each corresponding key
             # using affine fetched from d[affine_key]
-            d[key], affine_, new_affine = self.spacing_transform(
-                data_array=d[key], original_affine=affine, interp_order=interp)
-            if d.get(original_key, None) is None:
-                # set the 'original_affine' field
-                d[original_key] = affine_
+            d[key], _, new_affine = self.spacing_transform(
+                data_array=d[key], original_affine=d[affine_key], interp_order=interp)
```

Comment:
I think you should change the argument name of spacing from "original_affine" to "affine" as it's confusing with the real "original_affine" in dict.

Reply:
that's a good point, I've revised all cases